packages/react-lowcode/src/codegen/generation/generators/localization/messages-generator.ts

```diff
@@ -0,0 +1,7 @@
+export function addMessage(definitionSource: string, key: string, value: string): string | undefined {

```

Comment:
We already have function that writes to a localization file: https://github.com/iteria-app/lowcode/pull/125/files#diff-cefc92b1e2de6ee437d575a789cf94f79977a3be74facc01de4f46420aa29197R65

I like the automated test. I would like to see also escaped value.